ci-check-using-std.sh

```diff
@@ -19,7 +19,15 @@ usethis=$(grep --no-filename 'using std::' "$reporoot/src/"*.cpp \
     | sort -u)
 
 # Fall back to hardcoded list until the rule is actually enforced:
-usethis="string vector"
+usethis=" \
+    make_shared \
+    make_unique \
+    shared_ptr \
+    string \
+    unique_ptr \
+    vector \
+    weak_ptr \
+    "
```

Comment:
You don't need the `\` here because bash is waiting for the final `"` anyway.

Reply:
True, thanks. But actually... let me just turn this into a whitelist instead. There's not too many symbols left.

Reply:
Is also fine for me!